src/Azure.Functions.Cli/Azure.Functions.Cli.csproj

```diff
@@ -117,9 +117,9 @@
     <PackageReference Include="Microsoft.AspNetCore.DataProtection" Version="2.2.0" />
     <PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage.Internal" Version="1.4.0" />
     <PackageReference Include="Microsoft.Azure.Functions.JavaWorker" Version="1.5.2-SNAPSHOT" />
-    <PackageReference Include="Microsoft.Azure.Functions.NodeJsWorker" Version="1.1.1" />
+    <PackageReference Include="Microsoft.Azure.Functions.NodeJsWorker" Version="1.2.0" />
```

Comment:
@mhoeger fyi, I saw you updated the Node worker version in the host. Updating this as part of the release

Reply:
@ankitkumarr - you da best! thanks!